Jeff_pet_codes/ArrayThings/src/Student.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * Created by Jeffries Pc on 21/07/2017.
+ */
+public class Student {
+
+    private ArrayList<Integer> createStudentScores() {
+        ArrayList<Integer> studentScores = new ArrayList<>();
+
+        studentScores.add(70);
+        studentScores.add(77);
+        studentScores.add(75);
+        studentScores.add(99);
+        studentScores.add(55);
+        studentScores.add(55);
+        studentScores.add(87);
+        studentScores.add(78);
+
+        return studentScores;
+
+
+    }
+
+    public void assignStudentScore() {
```

Comment:
The method name does not reflect what the code content is doing. Method name says, assign student score and this is the only thing it needs to do and not going on to print things out. Take note.